opentelemetry-jaeger/src/lib.rs

```diff
@@ -431,7 +430,8 @@ impl PipelineBuilder {
     /// Build a configured `sdk::trace::TracerProvider` with the recommended defaults.
     pub fn build(
         mut self,
-    ) -> Result<sdk::trace::TracerProvider, Box<dyn Error + Send + Sync + 'static>> {
+    ) -> Result<sdk::trace::TracerProvider, Box<dyn std::error::Error + Send + Sync + 'static>>
```

Comment:
These could be `TraceError` too right? Would be nice to get the box dyn errors out of the api where possible